runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -10730,16 +10675,11 @@ J9::Z::TreeEvaluator::VMnewEvaluator(TR::Node * node, TR::CodeGenerator * cg)
    TR_OpaqueClassBlock * classAddress = 0;
    TR::Register *classReg              = NULL;
    TR::Register *resReg                = NULL;
-   TR::Register *zeroReg               = NULL;
    TR::Register *litPoolBaseReg        = NULL;
    TR::Register *enumReg               = NULL;
    TR::Register *copyReg               = NULL;
-   TR::Register *classRegAOT           = NULL;
    TR::Register *temp1Reg              = NULL;
```

Comment:
As you are using temp1Reg to hold the data size of the array / object, can we rename it to make code more readable ?